staging/src/k8s.io/client-go/tools/leaderelection/leaderelection_test.go

```diff
@@ -255,15 +266,18 @@ func testTryAcquireOrRenew(t *testing.T, objectType string) {
 				Identity:      "baz",
 				EventRecorder: &record.FakeRecorder{},
 			}
-			c := &fakecorev1.FakeCoreV1{Fake: &core.Fake{}}
+			fake := &core.Fake{}
```

Comment:
Actually, I suspect you just want to use:

https://godoc.org/k8s.io/client-go/kubernetes/fake#Clientset

which embeds a testing.Fake and implements clientset.Interface.

Reply:
yes, much better now!